core/libp2p/network/impl/connection_manager_impl.hpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CONNECTION_MANAGER_IMPL_HPP
+#define KAGOME_CONNECTION_MANAGER_IMPL_HPP
+
+#include "libp2p/network/connection_manager.hpp"
+#include "libp2p/network/transport_manager.hpp"
+#include "libp2p/peer/address_repository.hpp"
+#include "libp2p/peer/peer_id.hpp"
+
+namespace libp2p::network {
+
+  class ConnectionManagerImpl : public ConnectionManager {
+   public:
+    ConnectionManagerImpl(
+        std::shared_ptr<peer::AddressRepository> addrRepo,
```

Comment:
```suggestion
        std::shared_ptr<peer::AddressRepository> addr_repo,
```
And the one below

Reply:
it is not a member, just a variable

Reply:
Just variables are to be named in snake case according to Google Style, too.